frontend/catalyst/pennylane_extensions.py

```diff
@@ -2555,6 +2560,42 @@ def _get_batch_size(args_flat, axes_flat, axis_size):
     return batch_size
 
 
+class CallbackClosure:
+    """This is just a class containing data that is important for the callback."""
+
+    def __init__(self, *absargs, **abskwargs):
+        self.absargs = absargs
```

Comment:
Can we just call that args and kwargs?

Reply:
I like to denote that these are abstract arguments and abstract kwargs as opposed to concrete arguments and concrete kwargs.